src/Okta.Sdk/SessionsClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Sdk
+{
+    public sealed partial class SessionsClient : OktaClient, ISessionsClient, IAsyncEnumerable<ISession>
```

Comment:
Since there is no way to list all sessions (like you can with UsersClient or LogsClient), I don't think `SessionsClient` needs to be an `IAsyncEnumerable<ISession>`. That should mean that this whole file can be removed.

Reply:
In other words, unlike UsersClient and LogsClient, this is not possible:

```csharp
var sessions = await client.Sessions.GetSessions().ToArray();
```

so we don't need the `ISessionsClient` to be an enumerable.